Source/ASTextNode2.mm

```diff
@@ -313,6 +330,102 @@ - (UIAccessibilityTraits)defaultAccessibilityTraits
   return UIAccessibilityTraitStaticText;
 }
 
+static void ASUpdateAccessibilityFrame(ASTextNodeAccessiblityElement *accessibilityElement, ASTextLayout *layout, ASDisplayNode * _Nullable containerNode, ASDisplayNode *node) {
```

Comment:
Like your other functions, add a brief comment summarizing what this function does.